RELEASE_NOTES.md

```diff
@@ -8,6 +8,42 @@ autoconf   | >=2.61
 zlib1g-dev | >=0.13.x
 libtool    | >=2.x
 
-New Features
+This release integrates changes which were merged upstream, but after the **Felmingham v5.1.4**
```

Comment:
Should we have this kind of release "summary" at the start of the release notes? Where it is at the moment it looks like it is in the dependencies.